### PR TITLE
14.Ansible.Workshop

### PR DESCRIPTION
**Pull request by MsWik**
Please check

### CODE REVIEW

File: biaspaltsau_a/14.Ansible.Workshop/hosts.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        ansible_ssh_common_args: '-o ProxyCommand="ssh -W %h:%p -q jump_sa@178.124.206.53"'
+      hosts:
+        redmine:
+          ansible_host: 192.168.201.2
+          ansible_user: root
+
+
+
+
+jump:
```
Comment (pluhin) [inline biaspaltsau_a/14.Ansible.Workshop/hosts.yaml:12]: This is needlessly, please remove.
in this play we are doing nothing on jump server

File: biaspaltsau_a/14.Ansible.Workshop/redmine.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+  
+---
+- hosts: redmine
+  become: yes
+  pre_tasks:
+    - name: Redmine. Install packages
```
Comment (pluhin) [inline biaspaltsau_a/14.Ansible.Workshop/redmine.yaml:6]: This pre-task should be inside application role, please move

File: biaspaltsau_a/14.Ansible.Workshop/roles/db/tasks/main.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    name: "{{ app_db_name }}"
+    encoding: utf8
+
+- name: Configuring user priviliges
+  mysql_user:
+    name: User name "{{ app_db_user }}"
+    password: "{{ app_db_pass }}"
+    priv: "{{ app_db_name }}.*:ALL"
+
+- name: Config database
```
Comment (pluhin) [inline biaspaltsau_a/14.Ansible.Workshop/roles/db/tasks/main.yaml:24]: This is task which belongs to application role, please move

File: biaspaltsau_a/14.Ansible.Workshop/roles/db/templates/my.cnf.j2 [hunk 4]
```diff
@@ -0,0 +1,5 @@
+[client]
```
Comment (pluhin) [inline biaspaltsau_a/14.Ansible.Workshop/roles/db/templates/my.cnf.j2:1]: I do not remember this in our workshop, please remove

File: biaspaltsau_a/14.Ansible.Workshop/roles/redmine_all/tasks/main.yaml [hunk 5]
```diff
@@ -0,0 +1,105 @@
+---
```
Comment (pluhin) [inline biaspaltsau_a/14.Ansible.Workshop/roles/redmine_all/tasks/main.yaml:1]: wrong dividing of roles, you have all tasks in all roles, there are a lot of duplication code

**Review by pluhin — CHANGES_REQUESTED** [review]

File: biaspaltsau_a/14.Ansible.Workshop/hosts.yaml (hunk 1 above)
Comment (MsWik) [reply biaspaltsau_a/14.Ansible.Workshop/hosts.yaml]: corrected

File: biaspaltsau_a/14.Ansible.Workshop/roles/db/tasks/main.yaml (hunk 3 above)
Comment (MsWik) [reply biaspaltsau_a/14.Ansible.Workshop/roles/db/tasks/main.yaml:35]: corrected

File: biaspaltsau_a/14.Ansible.Workshop/redmine.yaml (hunk 2 above)
Comment (MsWik) [reply biaspaltsau_a/14.Ansible.Workshop/redmine.yaml]: corrected

File: biaspaltsau_a/14.Ansible.Workshop/roles/db/templates/my.cnf.j2 (hunk 4 above)
Comment (MsWik) [reply biaspaltsau_a/14.Ansible.Workshop/roles/db/templates/my.cnf.j2]: corrected

File: biaspaltsau_a/14.Ansible.Workshop/roles/redmine_all/tasks/main.yaml (hunk 5 above)
Comment (MsWik) [reply biaspaltsau_a/14.Ansible.Workshop/roles/redmine_all/tasks/main.yaml:1]: corrected

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**